package/MDAnalysis/analysis/base.py

```diff
@@ -402,20 +712,43 @@ def run(self, start=None, stop=None, step=None, frames=None,
         step : int, optional
             number of frames to skip between each analysed frame
         frames : array_like, optional
-            array of integers or booleans to slice trajectory; `frames` can
-            only be used *instead* of `start`, `stop`, and `step`. Setting
-            *both* `frames` and at least one of `start`, `stop`, `step` to a
-            non-default value will raise a :exc:`ValueError`.
+            array of integers or booleans to slice trajectory; ``frames`` can
+            only be used *instead* of ``start``, ``stop``, and ``step``. Setting
+            *both* ``frames`` and at least one of ``start``, ``stop``, ``step``
+            to a non-default value will raise a :exc:`ValueError`.
 
             .. versionadded:: 2.2.0
-
         verbose : bool, optional
             Turn on verbosity
 
         progressbar_kwargs : dict, optional
             ProgressBar keywords with custom parameters regarding progress bar
-            position, etc; see :class:`MDAnalysis.lib.log.ProgressBar` for full
-            list.
+            position, etc; see :class:`MDAnalysis.lib.log.ProgressBar`
+            for full list. Available only for ``backend='serial'``
+
+            .. versionadded:: 2.8.0
+        backend : Union[str, BackendBase], optional
+            By default, performs calculations in a serial fashion.
+            Otherwise, user can choose a backend: ``str`` is matched to a
+            builtin backend (one of ``serial``, ``multiprocessing`` and
+            ``dask``), or a :class:`MDAnalysis.analysis.results.BackendBase`
+            subclass.
+
```

Comment:
add missing 
```suggestion

            .. versionadded:: 2.8.0
```
(might have been accidentally added one above)